src/swig/test/python/test_projdata_profiles.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import pytest
+
+from examples.python.plot_projdata_profiles import compress_and_extract_1d_from_nd_array
```

Comment:
If this pytest fails, it will likely be here. This worked on my debugger but failed on my CLI, it was probably an enviroment issue.

Reply:
easiest silly work-around is probably
```suggestion
import os, sys
sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))))
from examples.python.plot_projdata_profiles import compress_and_extract_1d_from_nd_array
```